canonical-websites/www.ubuntu.com
# Transition Autopilot from the cloud overview page

## Done
Update the cloud overview page to replace autopilot with conjure-up.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/cloud](http://0.0.0.0:8001/cloud)
- Check that the content matches the [copy doc](https://docs.google.com/document/d/1mQjVXDcoP8ChOzRdmuI55BdXhbURD53qBNc2bgJhfhg/edit#)
- Check on different screen sizes

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/2187
